index.html

```diff
@@ -159,141 +156,401 @@ <h2>
+          In the [=pointer lock state=] the system mouse cursor must be hidden.
+          Movement and button presses of the mouse must not cause the window to
+          lose focus.
+        </p>
+        <p class="note">
+          Synthetic mouse events created by application script act the same
+          regardless of lock state.
+        </p>
+      </section>
+      <section data-dfn-for="PointerLockOptions">
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl>
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `pointerlockchange` and `pointerlockerror` Events
+        </h3>
+        <p>
+          Two events are used to communicate pointer lock state change or an
+          error in changing state. They are named <dfn>pointerlockchange</dfn>
+          and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
+          for any reason a {{Document/pointerlockchange}} event must be sent.
+        </p>
+        <p>
+          The [=user agents=] must deliver these events by <a data-lt=
+          "queue a task">queuing a task</a> to [=fire an event=] of the
+          appropriate name with its `bubbles` attribute [[DOM]] set to false to
+          the pointer lock [=Event/target=] element's [=Node/node document=].
+        </p>
+        <p class="note">
+          Magnification software increases the size of content on the screen.
+          It uses the mouse to move the magnified point of focus around. When a
+          pointer lock is initiated, the magnification software needs to switch
+          to using the keyboard to move the magnified point of focus around
+          instead. When a {{pointerlockchange}} event is fired, web browsers
+          therefore need to make sure the event is communicated to assistive
+          technologies like screen magnifiers.
+        </p>
+      </section>
+      <section>
+        <h3>
+          <dfn>Exit Pointer Lock</dfn>
+        </h3>
+        <p>
+          The process of exiting pointer lock, given an element, is as follows:
+        </p>
+        <ol>
+          <li>The system mouse cursor must be displayed again and positioned at
+          [=cursor position=]
+          </li>
```

Comment:
Was "cursor position" saved somewhere in the spec? It should be saved on the element as an internal slot maybe?  

```suggestion
          <li>Display the system mouse cursor  again and position it at [=cursor position=].
          </li>
```

Reply:
I wonder would it be duplicate to existing screenX / screenY attributes in MouseEvent?